PIL/Image.py

```diff
@@ -1814,6 +1893,7 @@ def transform(x, y, matrix):
         return self.transform((w, h), AFFINE, matrix, resample)
 
     def save(self, fp, format=None, **params):
+        # type: (Union[Text, pathlib.Path, BytesIO], Optional[Text], **Any) -> None
```

Comment:
I think this is resolved by using `BinaryIO`, which is defined in `typing`; I've come across this before while trying to resolve an issue in Zulip. Also see [stack overflow](https://stackoverflow.com/questions/41748896/the-correct-way-to-annotate-a-file-type-in-python).
I've made this change in an upcoming set of commits.